doc/nrf/device_guides/working_with_nrf/nrf54l/ug_nrf54l15_gs.rst

```diff
@@ -56,299 +54,13 @@ Complete the following steps to test if the PDK works correctly:
 
 If something does not work as expected, contact Nordic Semiconductor support.
 
-.. _nrf54l15_gs_installing_software:
+.. _ug_nrf54l15_gs_installing_software:
 
 Installing the required software
 ********************************
```

Comment:
Not really sure if this section is needed. We're mentioning the available terminal emulators in https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/test_and_optimize.html .
Maybe just:
```suggestion
Additional software
********************

You will need a :ref:`terminal emulator <test_and_optimize>` to program the sample and read the logs.
The recommended emulators are `nRF Connect Serial Terminal`_ or the nRF Terminal (part of the `nRF Connect for Visual Studio Code`_ extension).

.. note::
   The |nRFVSC| GUI does not yet support the nRF54L15.

```